tests/func_test.py

```diff
@@ -0,0 +1,65 @@
+from xdsl.ir import Block, Region
+from xdsl.dialects.func import FuncOp, Return
+from xdsl.dialects.arith import Addi, Constant
+from xdsl.dialects.builtin import IntegerAttr, i32
+
+
+def test_func():
+    # Create two constants and add them, add them in a region and
+    # create a function
+    a = Constant.from_attr(IntegerAttr.from_int_and_width(1, 32), i32)
```

Comment:
You might want to use `from_int_constant` here.

Reply:
Changed the first tests, but left the others to have coverage